test/functional/p2p_headers_sync_with_minchainwork.py

```diff
@@ -48,6 +50,10 @@ def reconnect_all(self):
         self.connect_nodes(0, 2)
         self.connect_nodes(0, 3)
 
+    def mocktime_all(self, time):
+        for n in self.nodes:
+            n.setmocktime(time)
+
```

Comment:
Setting mock time for all nodes seems like it could potentially be useful in multiple functional tests.  Maybe it could be added to `BitcoinTestFramework` rather than live in this functional test?

Reply:
That's reasonable. I don't feel super strongly about it.